concrete/src/File/Type/Type.php

```diff
@@ -421,7 +421,7 @@ public function getThumbnail($fullImageTag = true)
      */
     public function supportsThumbnails()
     {
-        if ($this->getName() == 'PNG' || $this->getName() == 'JPEG') {
+        if ($this->getName() == 'PNG' || $this->getName() == 'JPEG'|| $this->getName() == 'GIF') {
```

Comment:
FWIW I'd only call `getName` once and just use a variable. I'm not familiar with this bit, is `->getName()` guaranteed to be uppercased?